plugin/trino-hudi/src/test/resources/hudi-testing-data/hudi_custom_keygen_pt_v8_mor.md

```diff
@@ -0,0 +1,65 @@
+## Create script
+
+Structure of table:
+- MOR table with CustomKeyGenerator
+- Revision: f7157954a9819137446d8e3a1d331d003f069414
+- No log files
+
+
+```scala
+  test("Create MOR table with custom keygen partition field") {
+    withTempDir { tmp =>
+        val tableName = "hudi_custom_keygen_pt_v8_mor"
+
+        spark.sql(
+            s"""
+               |CREATE TABLE $tableName (
+               |  id INT,
+               |  name STRING,
+               |  price DOUBLE,
+               |  ts LONG,
+               |  -- Partition Source Fields --
+               |  part_country STRING,
+               |  part_date BIGINT
+               |) USING hudi
+               | LOCATION '${tmp.getCanonicalPath}'
+               | TBLPROPERTIES (
+               |  primaryKey = 'id',
+               |  type = 'mor',
+               |  preCombineField = 'ts',
+               |  -- Timestamp Keygen and Partition Configs --
+               |  hoodie.table.keygenerator.class = 'org.apache.hudi.keygen.CustomKeyGenerator',
+               |  hoodie.datasource.write.partitionpath.field = 'part_country:SIMPLE,part_date:TIMESTAMP',
+               |  hoodie.keygen.timebased.timestamp.type = 'EPOCHMILLISECONDS',
+               |  hoodie.keygen.timebased.output.dateformat = 'yyyy-MM-dd',
+               |  hoodie.keygen.timebased.timezone = 'UTC'
+               | ) PARTITIONED BY (part_country, part_date)
+     """.stripMargin)
+
+        // To not trigger compaction scheduling, and compaction
+        spark.sql(s"set hoodie.compact.inline.max.delta.commits=9999")
+        spark.sql(s"set hoodie.compact.inline=false")
+
+        // Configure Hudi properties
+        spark.sql(s"SET hoodie.metadata.compact.max.delta.commits=1")
```

Comment:
Should this be removed too to allow log files?

Reply:
No updates on this table, so all data is written to parquet or updates the parquet file.

No logs will be generated.